bin/action_subscriber

```diff
@@ -35,6 +35,8 @@ module ActionSubscriber
         ::ActionSubscriber::Babou.auto_pop!
       when /subscribe/i then
         ::ActionSubscriber::Babou.start_subscribers
+      else
+        puts "ActionSubscriber.configuration.mode must be 'pop' or 'subscribe'.  Currently set to '#{::ActionSubscriber.configuration.mode}'"
```

Comment:
We should probably `fail` here instead of logging to stdout. I would find that more helpful than a log message. #MyOpinion